src/components/signup/components/SignupForm.js

```diff
@@ -45,30 +44,18 @@ class SignupModal extends React.Component {
                 title={TITLE}
                 okText={TITLE}
                 onCancel={onCancel}
-                onOk={onCreate}
+                onOk = {onCreate}
```

Comment:
eric can you remove the space before and after the equal sign - `onOk={onCreate}`